scheduler.go

```diff
@@ -299,7 +299,7 @@ func (s *Scheduler) beat() {
 		entries = append(entries, e)
 	}
 	s.logger.Debugf("Writing entries %v", entries)
-	if err := s.rdb.WriteSchedulerEntries(s.id, entries, 5*time.Second); err != nil {
+	if err := s.rdb.WriteSchedulerEntries(s.id, entries, 10*time.Second); err != nil {
```

Comment:
should we set it as const instead ?

Reply:
This is not a regression, it makes sense to me to follow the pattern.

Constants are useful of course, especially when there is an established pattern, otherwise it's mostly an indirection that slightly reduces readability.

Reply:
@linhbkhn95 My bad, it _is_ already the pattern: 
https://github.com/hibiken/asynq/blob/c97652d408e04eeb944e7ec87cddaae77a5780aa/server.go#L415-L419